examples/examples_go_test.go

```diff
@@ -19,6 +18,8 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"os"
```

Comment:
`"os"` should be in the top block of imports.

Reply:
Have you automated ordering imports yet? I use C-c C-a for go-import-add but it does not do the right thing. On save I use gofmt. Perhaps there's something stronger than gofmt that could reorder them the way we need to on save?

Reply:
I have automated sorting them within blocks, but I have not automated sorting between blocks. I generally add imports manually, sometimes `eglot` does automatic insertion. I run `gofmt` on save, but that only alphabetizes, not sorting into blocks.

Reply:
https://github.com/mvdan/gofumpt is the stronger version. You could try `(setq gofmt-command "gofumpt")`.